spec/features/editing_tags/edit_tags_spec.rb

```diff
@@ -16,19 +17,21 @@
   end
 
   def given_there_is_an_edition
-    document_type = build(:document_type, tags: [multi_tag_field, single_tag_field])
-
-    tag_linkables = [initial_tag, tag_to_select_1, tag_to_select_2]
-    stub_publishing_api_has_linkables(tag_linkables, document_type: "world_location")
-    stub_publishing_api_has_linkables(tag_linkables, document_type: "organisation")
+    news_story_tags = DocumentType.find("news_story").tags
```

Comment:
This seems to be a new problem, as our content fields got covered implicitly by the format specs, which no longer touch this page. Is there a way we could do this that doesn't rely on a document type? A single document type is unlikely to have all of the possible tags, so ideally we want the spec to cover all kinds; one approach would be to list them all explicitly here.

However, it's a bit strange to have a feature spec that only makes use of some of its setup. It may be unclear why all of the news tags are necessary to someone reading this in future. One approach might be to make the spec exercise all the different tags, which means we're thoroughly testing them at the integration level, albeit with more complexity. Another approach could be to use the view spec for the tags edit page, to check each one renders with appropriate elements.

Although we seem to be pushing for the latter approach with [the work on content fields](https://github.com/alphagov/content-publisher/blob/master/spec/features/editing_content/edit_edition_spec.rb), it turns out other specs [sneakily ensure we test all the content fields](https://github.com/alphagov/content-publisher/blob/master/spec/features/workflow/create_document_spec.rb#L35). So it does look like testing all the tags via this feature is the way to go.

Reply:
Have now changed this to `all_tags = DocumentType.all.flat_map(&:tags).uniq(&:class)` 👍